src/poliastro/twobody/orbit.py

```diff
@@ -297,6 +297,46 @@ def circular(
             attractor, a, ecc, inc, raan, argp, arglat, epoch, plane
         )
 
+    @classmethod
+    @u.quantity_input(angular_velocity=u.rad / u.s, period=u.s, hill_radius=u.m)
+    def geostationary(
+        cls, attractor, angular_velocity=None, period=None, hill_radius=None
+    ):
+        """Return the geostationary orbit for the given attractor and its rotational speed.
+
+        Parameters
+        ----------
+        attractor : Body
+            Main attractor.
+        angular_velocity : ~astropy.units.Quantity
+            Rotational angular velocity of the attractor.
+        period : ~astropy.units.Quantity
+            Attractor's rotational period, ignored if angular_velocity is passed.
+        hill_radius : ~astropy.units.Quantity
+            Radius of Hill sphere of the attractor (optional). Gravitational sphere of
+            influence of parent body is ignored if hill_radius is not provided.
+        """
+
+        if angular_velocity is None and period is None:
+            raise ValueError(
+                "At least one among angular_velocity or period must be passed"
+            )
+
+        if angular_velocity is None:
+            angular_velocity = 2 * np.pi / period
+
+        # Find out geostationary radius using r = cube_root(GM/(angular velocity)^2)
+        with u.set_enabled_equivalencies(u.dimensionless_angles()):
+            geo_radius = np.cbrt(attractor.k / np.square(angular_velocity.to(1 / u.s)))
+
+        if hill_radius is not None and geo_radius > hill_radius:
```

Comment:
> Very nice addition!

Thanks :)

> Do you have any documentation on why to use the Hill radius instead of the Laplace radius? 

Hill sphere/radius makes more sense to me here as we are looking for a stable orbit (explained here  https://astronomy.stackexchange.com/a/7693/25384). However [this ](https://space.stackexchange.com/a/3940)is more elaborated one.

> Just curious, it would be great to add it to the docstring.

Sure, will add that.

Reply:
That's right, the Hill sphere makes more sense in this context. I also see a comment of mine in the second answer :stuck_out_tongue:

Reply:
oh yes  👍